mmv1/third_party/terraform/services/logging/resource_logging_bucket_config.go

```diff
@@ -287,6 +314,10 @@ func resourceLoggingBucketConfigRead(d *schema.ResourceData, meta interface{}) e
 		return fmt.Errorf("Error setting cmek_settings: %s", err)
 	}
 
+	if err := d.Set("index_configs", flattenIndexConfigs(res["indexConfigs"].([]interface{}))); err != nil {
```

Comment:
I suggest breaking this line up and first checking that `res["indexConfigs"]` is, in fact, a slice of `any` (or `interface{}`).

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/9109#discussion_r1367251751 instead- the stanza we use in generated code like subnetwork is resilient to as many edge cases as possible.